content/08-cardano-sidechains/03-example-evm-sidechain/04-deploy-smart-contracts/02-setup-development.mdx

```diff
@@ -0,0 +1,294 @@
+888    888                      888 888               888
+8888888888  8888b.  888d888 .d88888 88888b.   8888b.  888888
+888    888     "88b 888P"  d88" 888 888 "88b     "88b 888
+888    888 .d888888 888    888  888 888  888 .d888888 888
+888    888 888  888 888    Y88b 888 888  888 888  888 Y88b.
+888    888 "Y888888 888     "Y88888 888  888 "Y888888  "Y888
+
+👷 Welcome to Hardhat v2.9.9 👷‍
+
+? What do you want to do? …
+❯ Create a JavaScript project
+  Create a TypeScript project
+  Create an empty hardhat.config.js
+  Quit
+```
+
+<!-- Hardhat will ask you to install `hardhat-waffle` and `hardhat-ethers` which is required for tests built with Waffle. In case you've missed it, install them by executing the following:
+
+```
+npm install --save-dev @nomiclabs/hardhat-waffle ethereum-waffle chai @nomiclabs/hardhat-ethers ethers
+``` -->
+
+## Running tasks
+
+To view all options and commands with your Hardhat project, simply run `npx hardhat`:
+
+```
+$ npx hardhat
+Hardhat version 2.12.2
+
+Usage: hardhat [GLOBAL OPTIONS] <TASK> [TASK OPTIONS]
+
+GLOBAL OPTIONS:
+
+  --config           	A Hardhat config file
+  --emoji            	Use emoji in messages
+  --flamegraph       	Generate a flamegraph of your Hardhat tasks 
+  --help             	Shows this message, or a task's help if its name is provided 
+  --max-memory       	The maximum amount of memory that Hardhat can use
+  --network          	The network to connect to
+  --show-stack-traces	Show stack traces (always enabled on CI servers)
+  --tsconfig         	A TypeScript config file
+  --typecheck        	Enable TypeScript type-checking of your scripts/tests 
+  --verbose          	Enables Hardhat verbose logging 
+  --version          	Shows hardhat's version
+
+
+AVAILABLE TASKS:
+
+  check             	Check whatever you need
+  clean             	Clears the cache and deletes all artifacts
+  compile           	Compiles the entire project, building all artifacts
+  console           	Opens a Hardhat console
+  coverage          	Generates a code coverage report for tests
+  flatten           	Flattens and prints contracts and their dependencies
+  gas-reporter:merge	
+  help              	Prints this message
+  node              	Starts a JSON-RPC server on top of the Hardhat network
+  run               	Runs a user-defined script after compiling the project
+  test              	Runs mocha tests
+  typechain         	Generates Typechain typings for compiled contracts
+  verify            	Verifies contract on Etherscan
+
+To get help for a specific task run: npx hardhat help [task]
+```
+
+If you need additional information on using Hardhat, see official [Hardhat documentation](https://hardhat.org/getting-started/).
+
+## Step 02 - Configure Hardhat
+
+Our sample JavaScript project contains a `hardhat.config.js` file. The `networks` config field is an optional object where network names map to their configuration and we can set `accounts` we want to sign transactions. For example:
```

Comment:
The IOG style guide recommends avoiding 'we' - it is never clear who is included.
In this sentence, it works to change 'we' to 'you'.